### DISCUSSION TITLE
All setup done ,but its not starting

### DISCUSSION THREAD

**Discussion by vindevvindev** [MRColorR/money4band#77] [Q&A] [answered]
I have setup apps using OPTION 3
But when I try to run it using OPTION 5, I get 
`This menu item will launch all the apps using the configured .env file and the docker-compose.yaml file (Docker must be already installed and running)
Do you wish to proceed Y/N?: y
Error response from daemon: Minimum memory limit allowed is 6MB
All Apps started
You can visit the web dashboard on http://localhost:8081
If not already done, use the previously generated apps nodes URLs to add your device in any apps dashboard that require node claiming/registration (e.g. Earnapp, ProxyRack, etc.)
Now press enter to go back to the menu:` 
I have already docker installed and I can use docker without any issues.
after executing above command the DOCKER IMAGES start to appear on IMAGES tab ,but I can't see any CONTAINERS running.
I am getting error : `Error response from daemon: Minimum memory limit allowed is 6MB` but right after that it shows `All Apps started
You can visit the web dashboard on http://localhost:8081`
help me with this//

**Comment by MRColorR** [comment]
Hello, are you using the latest release? 
On which device are you running it?

The problem is either a missing 'm' at the end of one or more of the limits variables in the .env file generated or a value smaller than 6m but I think is the first case. Could you check?

**Reply by vindevvindev** [reply]
I checked .env file and this is what I found
`# PROJECT APPS LIMITS AND EXRA APPS LIMITS
# CPU limits are in % of a single core. On a 4 cores cpu 0.5=12,5%; 0.7=17,5%; 0.8=20%; 1=25%; 2=50%; 4=100% if you assign more than 1 core the app will be multicores aware
APP_CPU_LIMIT_LITTLE=0.8
APP_CPU_LIMIT_MEDIUM=1
APP_CPU_LIMIT_BIG=2
APP_CPU_LIMIT_HUGE=4

# MEM limits are in MB and the limit is 200% of the reservation
RAM_CAP_MB_DEFAULT=6144m
# Following values in the .env file are automatically calculated from RAM_CAP_MB_DEFAULT by the setup script
APP_MEM_RESERV_LITTLE=64m
APP_MEM_LIMIT_LITTLE=128m

APP_MEM_RESERV_MEDIUM=128m
APP_MEM_LIMIT_MEDIUM=256m

APP_MEM_RESERV_BIG=256m
APP_MEM_LIMIT_BIG=512m

APP_MEM_RESERV_HUGE=512m
APP_MEM_LIMIT_HUGE=1024m
`
what should I fix

**Reply by MRColorR** [reply]
file  looks good indeed ... so i don't know why this is happening just in your case. could you try again and see if the eror occurs again ?
-  on which device you are running it on? 
- runme.sh or runme.ps1 ?
- are you on latest docker version ? which version ?
- info: I saw from the community discord server logs that you tried to join. if you want to join the Discord community you need a better name. not repeated, a profile photo, etc. Since your account has been marked by Discord as suspicious the server does not accept your request.

**Reply by MRColorR** [reply]
@vindevvindev I've identified and addressed an issue in the runme.ps1 script, specifically in the functions related to computer information retrieval and RAM limits calculation. The problem was that the total RAM of the operating system was not being accurately retrieved. This has now been corrected in the release version 2.7.4.

try it and let me know

**Reply by vindevvindev** [reply]
I've i5 12500H  16GB Ram  Windows11 Home 
I ve run it in PowerShell using runme.ps1 in downloads directory
but docker is in main directory
i am running (Money4band-v.2.7.3)
my docker present info
`Client:
 Cloud integration: v1.0.35+desktop.5
 Version:           24.0.7
 API version:       1.43
 Go version:        go1.20.10
 Git commit:        afdd53b
 Built:             Thu Oct 26 09:08:44 2023
 OS/Arch:           windows/amd64
 Context:           default

Server: Docker Desktop 4.26.1 (131620)
 Engine:
  Version:          24.0.7
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.10
  Git commit:       311b9ff
  Built:            Thu Oct 26 09:08:02 2023
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.25
  GitCommit:        d8f198a4ed8892c764191ef7b3b06d8a2eeb5c7f
 runc:
  Version:          1.1.10
  GitCommit:        v1.1.10-0-g18a0cb0
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0`

**Reply by vindevvindev** [reply]
after running runme.ps1 this is what i get 
`MONEY4BAND AUTOMATIC GUIDED SETUP v2.7.3
Script is up to date.
----------------------------------------------
Join our Discord community for updates, help, and discussions: https://discord.com/invite/Fq8eeazBAD
----------------------------------------------
Detected OS type: Windows
Detected architecture: amd64
Docker amd64 image architecture will be used if the app's image permits it`

**Reply by MRColorR** [reply, marked as answer]
Let me know if 2.7.4 solve your problem

**Reply by vindevvindev** [reply]
is 2.7.4 available yet?
sry updated and ran it 

yessir, its running now 
2.7.4 solves it
